LDMP/metadata_dialog.py

```diff
@@ -0,0 +1,220 @@
+"""Metadata editing dialog for Trends.Earth QGIS plugin."""
+
+import os
+from pathlib import Path
+
+import qgis.core
+from qgis.PyQt import QtCore
+from qgis.PyQt import QtGui
+from qgis.PyQt import QtWidgets
+from qgis.PyQt import uic
+
+from . import tr
+
+Ui_DlgDatasetMetadata, _ = uic.loadUiType(
+    str(Path(__file__).parents[0] / "gui/DlgDatasetMetadata.ui")
+)
+
+ICON_PATH = os.path.join(os.path.dirname(__file__), 'icons')
+
+
+class DlgDatasetMetadata(QtWidgets.QDialog, Ui_DlgDatasetMetadata):
+
+    layer: qgis.core.QgsMapLayer
+    metadata: qgis.core.QgsLayerMetadata
+
+    def __init__(self, parent=None):
+        super(DlgDatasetMetadata, self).__init__(parent)
+        self.setupUi(self)
+
+        self.btn_add_address.setIcon(
+            QtGui.QIcon(os.path.join(ICON_PATH, 'symbologyAdd.svg'))
+        )
+        self.btn_remove_address.setIcon(
+            QtGui.QIcon(os.path.join(ICON_PATH, 'symbologyRemove.svg'))
+        )
+        self.btn_new_category.setIcon(
+            QtGui.QIcon(os.path.join(ICON_PATH, 'symbologyAdd.svg'))
+        )
+        self.btn_add_default_category.setIcon(
+            QtGui.QIcon(os.path.join(ICON_PATH, 'mActionArrowRight.svg'))
+        )
+        self.btn_remove_category.setIcon(
+            QtGui.QIcon(os.path.join(ICON_PATH, 'symbologyRemove.svg'))
+        )
+
+        self.layer = None
+        self.metadata = None
+        self.default_categories = [tr("Farming"), tr("Biota"),
```

Comment:
Assume these categories come from the ISO standard?

Reply:
They are coming from the document attached here https://github.com/ConservationInternational/trends.earth/issues/518#issuecomment-1017538060. And these are also standard categories used in the ISO standard.